src/pages/terraform/cloud-docs/[[...page]].tsx

```diff
@@ -0,0 +1,30 @@
+import { ReactElement } from 'react'
+import terraformData from 'data/terraform.json'
+import { Product } from 'types/products'
+import { getStaticGenerationFunctions } from 'layouts/sidebar-sidecar/server'
+import SidebarSidecarLayout from 'layouts/sidebar-sidecar'
+import DocsView from 'views/docs-view'
+
+const basePath = 'cloud-docs'
+const baseName = 'Cloud Docs'
+const product = terraformData as Product
+const productSlugForLoader = 'terraform-website'
+
+const additionalComponents = {}
```

Comment:
Is this needed or stubbed for future use? A comment may be useful if the latter is true. 😊